pkg/config/vars.go

```diff
@@ -142,6 +132,57 @@ func setKwokctlConfigurationDefaults(config *v1alpha1.KwokctlConfiguration) *v1a
 	}
 	conf.QuietPull = ptrVar(envs.GetEnvBoolWithPrefix("QUIET_PULL", *conf.QuietPull))
 
+
+	v1alpha1.SetObjectDefaults_KwokctlConfiguration(config)
```

Comment:
Hi @wzshiming, how about the `v1alpha1.SetObjectDefaults_KwokctlConfiguration`, this function should be called at first, or is there some sepcial order?

Reply:
`v1alpha1.SetObjectDefaults_KwokctlConfiguration` This function sets the default value for variables that are not affected by other variables, but it may affect other variables, try to put it a little above.

Reply:
> Hi @wzshiming, how about the `v1alpha1.SetObjectDefaults_KwokctlConfiguration`, this function should be called at first, or is there some sepcial order?

Look the code again, we should put it **at last**, only cannot find variables from env or flags and then set the default value via `v1alpha1.SetObjectDefaults_KwokctlConfiguration`, if not, the default value(set by ObjectDefaults) will override users's config. Please correct me if I miss something.

Reply:
Makes sense